MAUI/DataGrid/Getting-started.md

```diff
@@ -0,0 +1,341 @@
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.SortingMode=DataGridSortingMode.Single; 
+{% endhighlight %}
+{% endtabs %}
+
+Run the application and touch the header cell to sort the data and the following output will be displayed.
+ 
+
+
+Sorting can also be configured by adding the column to the `SfDataGrid.SortColumnDescription` collection as follows.
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid.SortColumnDescription>
+    <syncfusion:SortColumnDescription ColumnName="CustomerID" />
+</syncfusion:SfDataGrid.SortColumnDescription> 
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.SortColumnDescription.Add (new SortColumnDescription () { ColumnName = "CustomerID" });
+{% endhighlight %}
+{% endtabs %}
+
+## Selection
+
+The SfDataGrid allows selecting the row or rows by setting the `SfDataGrid.SelectionMode` property. You can set the `SfDataGrid.SelectionMode` property to single, multiple, single deselect, or none. Information about the row or rows selected can be tracked using the `SfDataGrid.SelectedRow` and `SfDataGrid.SelectedRows` properties.
+
+The selection operations can be handled with the help of the `SelectionChanging` and `SelectionChanged` events of the SfDataGrid.
+
+## Loading the SfDataGrid with customized height and width
+
+The SfDataGrid can be load with specific height and width inside different layouts using the `SfDataGrid.HeightRequest` and `SfDataGrid.WidthRequest` properties. 
+
+The following code example illustrates how this can be done.
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:GettingStarted;assembly=GettingStarted"
+             xmlns:syncfusion="clr-namespace:Syncfusion.Maui.DataGrid;assembly=Syncfusion.Maui.DataGrid" >
+
+     <ContentPage.BindingContext>
+        <local:OrderInfoRepository x:Name="viewModel" />
+    </ContentPage.BindingContext>
+    
+        <sfgrid:SfDataGrid x:Name="dataGrid"
+                           ItemsSource="{Binding OrderInfoCollection}"
+                           HeightRequest="290"
+                           WidthRequest="200"
+                           VerticalOptions="CenterAndExpand"
+                           HorizontalOptions="Center"/>
+    
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+public MainPage()
+{
+    InitializeComponent();
+    OrderInfoRepository = new OrderInfoRepository();
+    dataGrid = new SfDataGrid();
+    dataGrid.ItemsSource = viewModel.OrdersInfo;
+    dataGrid.HeightRequest = 290;
+    dataGrid.WidthRequest = 200;
+    dataGrid.VerticalOptions = LayoutOptions.CenterAndExpand;
+    dataGrid.HorizontalOptions = LayoutOptions.Center;
+    this.Content = dataGrid;
+}
+{% endhighlight %}
+{% endtabs %}
+
+The following screenshot shows how the SfDataGrid is loaded with specific height and width with VerticalOptions and HorizontalOptions.
```

Comment:
The following screenshot shows how the SfDataGrid is loaded with specific height and width with VerticalOptions and HorizontalOptions: